client.go

```diff
@@ -546,3 +593,35 @@ func (s *Client) LoopInTerms(ctx context.Context) (
 
 	return s.Server.GetLoopInTerms(ctx)
 }
+
+// assertLndVersionCompatible makes sure the detected lnd version is compatible
+// with our current version requirements.
+func assertLndVersionCompatible(actualVersion *verrpc.Version, expectedMajor,
```

Comment:
I can see this code being used elsewhere. Isn't this something to move into `LndServices`? There is already a `GetInfo` call being executed always.